tests/vmi_networking_test.go

```diff
@@ -756,11 +756,14 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					ipv4NetworkCIDR = api.DefaultVMCIDR
 				}
 				By("Checking ping to gateway")
-				pingVirtualMachine(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
+				err = testPingFromVMI(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
+				Expect(err).ToNot(HaveOccurred(), "VMI %s was not able to ping gateway address %s", serverVMI.Name, gatewayIPFromCIDR(ipv4NetworkCIDR))
 
 				By("Checking ping to google")
-				pingVirtualMachine(serverVMI, "8.8.8.8", "\\$ ")
-				pingVirtualMachine(clientVMI, "google.com", "\\$ ")
+				err = testPingFromVMI(serverVMI, "8.8.8.8", "\\$ ")
+				Expect(err).ToNot(HaveOccurred(), "VMI %s was not able to ping google DNS server", serverVMI.Name)
```

Comment:
I would specify the address so it would be clear dns name translation is not involved. "to ping google DNS server (8.8.8.8)"

Reply:
yes, makes more sense. Done